pkg/sidecar/sidecar.go

```diff
@@ -88,7 +98,16 @@ func Run(version string) int {
 		return 1
 	}
 
-	// run until terminated
+	// run grpc server until terminated
+	server, err := grpc.NewServer(rt.KubeClient, grpc.ServerConfig{
+		Port:        *port,
+		TLSCertFile: *tlsCert,
+		TLSKeyFile:  *tlsKey,
+	})
+	if err != nil {
+		klog.Fatalf("failed to start server: %v", err)
```

Comment:
Suggestion: `Failed to start GRPC server: %v`

We'll probably have an HTTP server in addition.